KatherineK/multi/apps/blogs_app/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.conf.urls import url
+from . import views
+urlpatterns = [
+    url(r'^$', views.index),
+    url(r'^new$', views.new),
+    url(r'^create$', views.create),
+    url(r'^(?P<number>\d+)$', views.display),
+    url(r'^(?P<number>\d+)/edit$', views.edit),

```

Comment:
This is great, I'm glad you're understanding route parameters. I would recommend switching the order on your url though. Mostly this is for convention, and we will talk about it more as we talk more about restful routing. 
url(r'^edit/(?P<number>\d+)$', views.edit),

The reason, as developers, that we prefer this structure is for readability in the url. Now it reads edit this blog, rather than id of blog to edit.